src/modules/colorPicker/ColorPickerUI/Program.cs

```diff
@@ -39,7 +39,7 @@ public static void Main(string[] args)
 
         private static void CurrentDomain_UnhandledException(object sender, UnhandledExceptionEventArgs e)
         {
-            Logger.LogError("Unhandled exception", (e.ExceptionObject is Exception) ? (e.ExceptionObject as Exception) : new Exception());
```

Comment:
The logic is changed. Perhaps this needs to be unrolled into an `if else` block instead, but we should keep logic here.

Reply:
`new Exception` raises the warning [CA2201: Do not raise reserved exception types](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2201).

My though process was that the `UnhandledExceptionEventArgs.ExceptionObject` will always be of type exception or null. If it is of type exception the code behaves the same. 

In the unexpected case that `UnhandledExceptionEventArgs.ExceptionObject` is not based on `Exception` or null I changed the code to pass `null` instead of `new Exception` into the `LogError(...)` Method.

Reply:
Sounds good to me, actually. Just verified LogError has that alternative signature. No idea why it wasn't used here in the first place 🤷 .
Thank you 😄